devel/ghidra/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           app 1.0
+
+name                ghidra
+homepage            https://ghidra-sre.org/
+master_sites        https://ghidra-sre.org/
+version             9.2.2
+use_zip             yes
+distfiles           ${name}_${version}_PUBLIC_20201229.zip
+checksums           rmd160  2b7a239329ba515010ff52725da08eb656f168f7 \
+                    sha256  8cf8806dd5b8b7c7826f04fad8b86fc7e07ea380eae497f3035f8c974de72cf8 \
+                    size    317805407
+revision            0
+
+categories          devel
+maintainers         {hexagonal-sun @hexagonal-sun} openmaintainer
```

Comment:
Sorry @hexagonal-sun, the format here still needs some tweaking:

The `@<name>` form is used for Github handles, which is correct here.
The plain `<name>` form is used for those who have a macports.org email address.
You may need to use the email address form, which would be `domain.com:you`:

Example:
```
maintainers  {sun.com:hexagonal @hexagonal-sun} openmaintainer
```

...if your email addressed looked something like `hexagonal@sun.com`.

Reply:
@herbygillot , thanks for the explanation! That should now be fixed.